docs/de/index.md

```diff
@@ -19,4 +19,4 @@ Die Installation erfolgt in vier einfachen Schritten:
   1. Entpacke das ZIP-Archiv und kopiere die 3 darin enthaltenen Datein auf die MicroSD-Karte (_**Formatiert in FAT32**_)
   1. Stecke die MicroSD-Karte in Deinen TXT und schalte ihn ein!
 
-Weitere Informationen findest Du in [unserem Wiki](https://github.com/ftCommunity/ftcommunity-TXT/wiki/%5BDE%5D-Anleitung%3A-Einrichtung-der-ftcommunity-TXT-Firmware) und im [Fischertechnik Community Forum](https://forum.ftcommunity.de/viewforum.php?f=33).
+Weitere Informationen findest Du in [unserem Wiki](http://cfw.ftcommunity.de/ftcommunity-TXT/de/getting-started/installation.html) und im [fischertechnik Community Forum](https://forum.ftcommunity.de/viewforum.php?f=33).
```

Comment:
"in [unserem Wiki]" -> "in der [Installations-Anleitung]" (der Link zeigt ja auch nicht mehr ins Wiki).